json/it.json

```diff
@@ -1,16 +1,16 @@
 {
     "A fresh verification link has been sent to your email address.": "Un nuovo link di verifica è stato inviato al tuo indirizzo email.",
-    "A new verification link has been sent to the email address you provided during registration.": "Un nuovo link di verifica è stato inviato all'indirizzo e-mail fornito durante la registrazione.",
+    "A new verification link has been sent to the email address you provided during registration.": "Un nuovo link di verifica è stato inviato all'indirizzo email fornito durante la registrazione.",
     "All rights reserved.": "Tutti i diritti riservati",
     "Already registered?": "Già registrato?",
-    "Before proceeding, please check your email for a verification link.": "Prima di continuare, controlla la tua posta elettronica per un nuovo link to verifica",
+    "Before proceeding, please check your email for a verification link.": "Prima di continuare, controlla la tua mail per il link di verifica",
     "Confirm Password": "Conferma Password",
     "E-Mail Address": "Indirizzo E-Mail",
```

Comment:
e-mail, mail, email... Why difference ?

Reply:
I want use "email" as much as possible for consistency. I'm changing "mail" with "casella email" because "posta elettronica" is an old terminology.

Reply:
Ok, but for the translation of "E-Mail Address", we have "Indirizzo E-Mail"

Reply:
yes, I was thinking of keeping "e-mail" when Laravel uses the hyphen